### ISSUE TITLE
"Part" text missing after using Bulk Edit

### ISSUE THREAD

**Issue by stevejburge** [publishpress/publishpress-series#453]
This may be related to https://github.com/publishpress/publishpress-series/issues/452

I used Bulk Edit to add posts to a Series:
<img width="1230" alt="Screen Shot 2022-03-08 at 1 26 29 PM" src="https://user-images.githubusercontent.com/3868028/157301995-e4a20a59-44f7-4d90-bcc4-3581675f0f3c.png">

**Comment by olatechpro** [comment]
@stevejburge i tried every method and trial to recreate this but wasn't able to. 

Can you confirm the series translation text wasn't modified in your setup?

That's the only last instance where you could have gotten the above result.

If you transalate Part %1$s of %2$s%6$s in the series using plugin like loco translate

**Comment by stevejburge** [comment]
@olatechpro yes, Loco Translate was on this site. I can't recreate this on another site

Let's consider this tested and drop it from the milestone for now
